In [1]:
items={1001:{"Name":"Rice","Price":67,"Quantity":100,"Expiry":None},
       1002:{"Name":"Dal","Price":57,"Quantity":110,"Expiry":None},
       1003:{"Name":"Sugar","Price":80,"Quantity":200,"Expiry":None},
       1004:{"Name":"Biscuit","Price":10,"Quantity":300,"Expiry":None},
       1005:{"Name":"Rajma","Price":50,"Quantity":100,"Expiry":None},
       1006:{"Name":"Shampoo","Price":150,"Quantity":100,"Expiry":2025},
       1007:{"Name":"Livon Serum","Price":250,"Quantity":100,"Expiry":2024},
       1008:{"Name":"Ketchup","Price":125,"Quantity":100,"Expiry":2023},
       1009:{"Name":"Wheat","Price":42,"Quantity":100,"Expiry":None},
       1010:{"Name":"Noodles","Price":55,"Quantity":100,"Expiry":None},
       1011:{"Name":"Pasta","Price":68,"Quantity":100,"Expiry":None},
       1012:{"Name":"Refined Oil","Price":110,"Quantity":100,"Expiry":None},
       1013:{"Name":"Hair oil","Price":70,"Quantity":100,"Expiry":2023},
       1014:{"Name":"Almonds","Price":200,"Quantity":100,"Expiry":None},
       1015:{"Name":"Raisins","Price":170,"Quantity":100,"Expiry":None},
       1016:{"Name":"Poha","Price":65,"Quantity":100,"Expiry":None},
       1017:{"Name":"Suji","Price":35,"Quantity":100,"Expiry":None},
       1018:{"Name":"Besan","Price":40,"Quantity":100,"Expiry":None},
       1019:{"Name":"Chole","Price":80,"Quantity":100,"Expiry":None},
       1020:{"Name":"Milk","Price":48,"Quantity":100,"Expiry":"2 days"},
       1021:{"Name":"Paneer","Price":250,"Quantity":100,"Expiry":"3 days"},
       1022:{"Name":"Icecream","Price":100,"Quantity":100,"Expiry":None},
       1023:{"Name":"Dark Chocolate","Price":67,"Quantity":100,"Expiry":2022},
       1024:{"Name":"Oregano","Price":80,"Quantity":100,"Expiry":2023},
       1025:{"Name":"Chilliflakes","Price":90,"Quantity":100,"Expiry":None},
       1026:{"Name":"Soyabean","Price":100,"Quantity":100,"Expiry":None},
       1027:{"Name":"Cheese","Price":15,"Quantity":100,"Expiry":2023},
       1028:{"Name":"Papad","Price":50,"Quantity":100,"Expiry":None},
       1029:{"Name":"Facewash","Price":100,"Quantity":100,"Expiry":2022},
       1030:{"Name":"Soap","Price":120,"Quantity":100,"Expiry":None}
       }

**Input details about the product you want to add in the inventory**

In [2]:
product_id=int(input("Enter product id: "))
name1=str(input("Enter product name: "))
pr=int(input("Enter product price: "))
quant=int(input("Enter product quantity: "))
expiry=int(input("Enter product expiry: "))


Enter product id: 1031
Enter product name: Toothpaste
Enter product price: 70
Enter product quantity: 100
Enter product expiry: 2023


**Adding the Item in the Inventory**

In [3]:
items[product_id]={"Name":name1,"Price":pr,"Quantity":quant,"Expiry":expiry}

**Displaying items in the dictionary to check whether the new product is added or not**

In [4]:
items

{1001: {'Expiry': None, 'Name': 'Rice', 'Price': 67, 'Quantity': 100},
 1002: {'Expiry': None, 'Name': 'Dal', 'Price': 57, 'Quantity': 110},
 1003: {'Expiry': None, 'Name': 'Sugar', 'Price': 80, 'Quantity': 200},
 1004: {'Expiry': None, 'Name': 'Biscuit', 'Price': 10, 'Quantity': 300},
 1005: {'Expiry': None, 'Name': 'Rajma', 'Price': 50, 'Quantity': 100},
 1006: {'Expiry': 2025, 'Name': 'Shampoo', 'Price': 150, 'Quantity': 100},
 1007: {'Expiry': 2024, 'Name': 'Livon Serum', 'Price': 250, 'Quantity': 100},
 1008: {'Expiry': 2023, 'Name': 'Ketchup', 'Price': 125, 'Quantity': 100},
 1009: {'Expiry': None, 'Name': 'Wheat', 'Price': 42, 'Quantity': 100},
 1010: {'Expiry': None, 'Name': 'Noodles', 'Price': 55, 'Quantity': 100},
 1011: {'Expiry': None, 'Name': 'Pasta', 'Price': 68, 'Quantity': 100},
 1012: {'Expiry': None, 'Name': 'Refined Oil', 'Price': 110, 'Quantity': 100},
 1013: {'Expiry': 2023, 'Name': 'Hair oil', 'Price': 70, 'Quantity': 100},
 1014: {'Expiry': None, 'Name': 'Almonds

**Product with product_id : 1031 has been added successfully**

**Adding the updation into the item_records.json file and using the same file item_records.json file in the purchasing program of the inventory management system**

In [5]:
import json
js_items=json. dumps(items)
fd=open("item_records.json","w")
fd.write(js_items)
fd.close()